realtime.js

```diff
@@ -145,3 +145,20 @@ exports.waittimes = function(times) {
         times: times
     });
 }
+
+// Takes in a list of tas to notify about their help time
+exports.notifytime = function(notif_tas) {
+    exports.seq = exports.seq + 1;
+    if (!sio) {
+        console.log("ERROR: Socket.io is not initialized yet");
+        return;
+    }
+
+    notif_tas.forEach(ta => {
```

Comment:
Instead of emitting one realtime message for each TA notification, how about emitting a single message with the entire array of notifications? That will reduce the number of messages being sent out in a short period of time.